src/lib/config.js

```diff
@@ -17,11 +17,16 @@ module.exports = {
       ACCOUNT_DETAILS: "/enter-account-details",
       CONFIRM_DETAILS: "/confirm-details",
       ABORT: "/abort",
+      CANNOT_PROCEED: "/cannot-proceed",
       DONE: "/done",
     },
     ANALYTICS: {
       DOMAIN: process.env.ANALYTICS_DOMAIN || "localhost",
     },
+    HAS_EXPIRY_DATE: {
```

Comment:
What is this being used for?

Reply:
That was just a carry over that got missed during initial setup of the repo, have removed